navigation/ca_hector/launch/default_mapping.launch

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
+
+<!-- NOTE:
+1. Maps created with Hector SLAM break with the create's odom shift:
+For the gazebo house environment, the maps created with Hector SLAM
+place the (0, 0) at the start position of the robot and not at Gazebo's
+(0,0). This will break when applying odom's shift for the create, as
+it will push the robot out of the map.
```

Comment:
Can you add a link to this PR? https://github.com/RoboticaUtnFrba/create_autonomy/pull/214

Reply:
Done!